source/Controlled.tsx

```diff
@@ -1,33 +1,57 @@
-import React, { StrictMode, memo, useCallback, useRef, useState } from 'react'
-import { bool, func, node, number, object, string } from 'prop-types'
 import 'focus-options-polyfill'
+import React, {
+  memo,
+  SFC,
+  StrictMode,
+  useCallback,
+  useRef,
+  useState
+} from 'react'
 import ControlledActivated from './ControlledActivated'
 import './styles.css'
 
-const Controlled = ({
+interface Props {
+  children: React.ReactNode
+  closeText?: string
+  isZoomed: boolean
+  onZoomChange?: (boolean) => void
+  openText?: string
+  overlayBgColorEnd?: string
+  overlayBgColorStart?: string
+  portalEl?: HTMLElement
+  scrollableEl?: HTMLElement | Window
+  transitionDuration?: number
+  wrapStyle?: React.CSSProperties | undefined
+  zoomMargin?: number
+  zoomZindex?: number
+}
+
+const Controlled: SFC<Props> = ({
   children,
-  closeText,
+  closeText = 'Unzoom image',
   isZoomed: isActive,
-  overlayBgColorEnd,
-  overlayBgColorStart,
+  overlayBgColorEnd = 'rgba(255, 255, 255, 0.95)',
+  overlayBgColorStart = 'rgba(255, 255, 255, 0)',
   portalEl,
   onZoomChange,
-  openText,
+  openText = 'Zoom image',
   scrollableEl,
-  transitionDuration,
-  wrapStyle,
-  zoomMargin,
-  zoomZindex
-}) => {
-  const [isChildLoaded, setIsChildLoaded] = useState(false)
-  const wrapRef = useRef(null)
-  const btnRef = useRef(null)
+  transitionDuration = 300,
+  wrapStyle = undefined,
```

Comment:
I think `undefined` is what this is if nothing is passed. The original proptype had it set to `null`, so I reckon we just don't give it `= undefined`